js/builder_graph.js

```diff
@@ -445,20 +445,34 @@ class PowderInputNode extends InputNode {
 
     compute_func(input_map) {
         // TODO: haha improve efficiency to O(n) dumb
-        // also, error handling is missing
         let input = this.input_field.value.trim();
         let powdering = [];
         let errorederrors = [];
         while (input) {
             let first = input.slice(0, 2);
             let powder = powderIDs.get(first);
             if (powder === undefined) {
-                return null;
+                if (first.length > 0)
+                    errorederrors.push(first);
```

Comment:
style: for here and below, wrap with `{}`

Reply:
Fixed.